packages/react-charting/src/components/DonutChart/DonutChart.base.tsx

```diff
@@ -126,8 +139,10 @@ export class DonutChartBase extends React.Component<IDonutChartProps, IDonutChar
                 focusedArcId={this.state.focusedArcId || ''}
                 href={this.props.href!}
                 calloutId={this._calloutId}
-                valueInsideDonut={this._toLocaleString(valueInsideDonut)}
+                valueInsideDonut={valueInsideDonut}
```

Comment:
why remove toLocaleString()

Reply:
toLocaleString() was used only to localize numbers like 39000 to 39,000
But now, it is not needed as we are formatting numbers with SI prefix like 39000 to 39.0k

Reply:
Is the SI prefix formatting applicable to valueInsideDonut as well?

Reply:
removed this change as discussed